src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  height: 100%;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  font-family: $main-font-family;
+  color: $main-color;
+
+  &__photo {
+    width: 100%;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 0 3px 40px 0;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $minor-color;
+    text-align: left;
+    margin: 0 0 16px;
+  }
+
+  &__feedback {
+    margin-bottom: 24px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    align-items: flex-end;
+  }
```

Comment:
The align-items property in the &__feedback block is not necessary. By default, flex items are stretched to fill the container. You can remove this property.